otter/integration/tests/test_scaling.py

```diff
@@ -123,3 +119,63 @@ def test_scaling_up(self):
         )
         return d
     test_scaling_up.timeout = 1800
+
+    def test_policy_execution_after_adding_clb(self):
```

Comment:
Oh ok.  Hmm... I guess just describe what this test is supposed to do, maybe plus either a TODO or a link to the issue?